### ISSUE TITLE
Custom exceptions should inherit from StandardError not Exception

### ISSUE THREAD

**Issue by masterkain** [gravityblast/musix_match#1]
Exception is too high level in the tree: http://whynotwiki.com/Ruby_/_Exception_handling#Exception_hierarchy

Currently when rescuing exceptions thrown by musix_match with `rescue => ex` will not trap anything.

**Comment by gravityblast** [comment]
Yes, I think the best way is to change all the custom exception and make them inherit from StandardError. what do you think?

**Comment by gravityblast** [comment]
you told me that in the title, I replied just after reading the body :) feel free to send me a pull request!
